app/views/estimate.scala.html

```diff
@@ -0,0 +1,84 @@
+@*
+ * Copyright 2018 HM Revenue & Customs
```

Comment:
The name of this file seems a bit confusing considering what already exists. It might be worth putting these new views in a separate directory within `app/views` to make it clear what they are.